gemrb/core/Game.cpp

```diff
@@ -40,8 +40,27 @@
 #include "GUI/GameControl.h"
 #include "System/DataStream.h"
 #include "System/StringBuffer.h"
+#include <iterator>
+#include <vector>
+#include <algorithm>
 
 namespace GemRB {
+	
+	
+struct HealingResource {
```

Comment:
I dont see why either of these 2 structs are declared here. Their use appears to be completely localized to `CastOnRest()`